dashboard/config/scripts_json/allthethings.script_json

```diff
@@ -3426,47 +4248,10 @@
     },
     {
       "chapter": 115,
-      "position": 10,
-      "activity_section_position": null,
-      "assessment": false,
-      "properties": {
-        "level_keys": [
-          "csp_socialBelonging_control",
-          "csp_socialBelonging_intervention"
-        ],
-        "variants": {
-          "csp_socialBelonging_intervention": {
-            "active": false,
-            "experiments": [
-              "socialBelongingTest"
-            ]
-          }
-        }
-      },
-      "bonus": false,
-      "seeding_key": {
-        "script_level.level_keys": [
-          "csp_socialBelonging_control",
-          "csp_socialBelonging_intervention"
-        ],
-        "lesson.key": "Swapped Levels",
-        "lesson_group.key": "",
-        "script.name": "allthethings"
-      },
-      "level_keys": [
-        "csp_socialBelonging_control",
-        "csp_socialBelonging_intervention"
-      ]
-    },
-    {
-      "chapter": 116,
```

Comment:
this block is disappearing because a previous PR removed the corresponding content from directly from  dashboard/config/scripts/allthethings.script, without updating the .script_json. normally the .script_json gets updated when the script edit page is used to make updates, such as on levelbuilder.